### PR TITLE
custom_fields: allow to search vocab

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes https://github.com/CERNDocumentServer/cds-rdm/issues/180
* closes https://github.com/CERNDocumentServer/cds-rdm/issues/143

image

### CODE REVIEW

**Comment by kpsherva** [comment]
the dropdown icon looks a little bit strange to me, what is it referring to?

**Comment by jrcastro2** [comment]
> the dropdown icon looks a little bit strange to me, what is it referring to?

These are custom fields, I added them without section name for now. We will need to find a proper name for this.

**Comment by kpsherva** [comment]
could we add a name? it looks broken otherwise

File: invenio_app_rdm/records_ui/views/filters.py [hunk 1]
```diff
@@ -182,6 +183,25 @@ def custom_fields_search(field, field_value):
     )
 
 
+def custom_vocab_fields_search(field, field_value, sub_field):
```
Comment (kpsherva) [inline invenio_app_rdm/records_ui/views/filters.py:186]: do we need a split from custom_fields_search? could we instead add a flag if is it a vocab and use a function above?

**Comment by jrcastro2** [comment]
> could we add a name? it looks broken otherwise, also the changes in react-invenio-forms are only to handle the fact we do not have a name?

Sure, name is temporal though
image

**Pull request merged**